src/plugins/custom_import_map/opensearch_dashboards.json

```diff
@@ -1,7 +1,7 @@
 {
     "id": "customImportMap",
-    "version": "2.0.0",
-    "opensearchDashboardsVersion": "opensearchDashboards",
+    "version": "2.2.0.0",
```

Comment:
Just want to double check here: is it supposed to be 2.2.0.0 or 2.2.0?

Reply:
It is `2.2.0.0`. opensearch-build reads these versions and if plugin version is not == `2.2.0.0` and osd version is not == `2.2.0`, it will fail